stock-filters/ChirpVillage/Builder.py

```diff
@@ -256,3 +282,23 @@ def construct(self, level, coords, door_coords, surface):
                 if x == maxx-1 or z == maxz-1 or x == minx or z == minz:
                     if x != (maxx+minx)/2 or z == maxz-1:
                         utilityFunctions.setBlock(level, hedge_block, surface.to_real_x(x), miny+1, surface.to_real_z(z))
+
+class Farm():
```

Comment:
I think we should rename either this or the construct_farm function. I can see the way it is now getting confusing

Reply:
I was trying to follow the builder naming scheme, but it might not be best for the farm. What did you want it to be named?

Reply:
Maybe FarmLot or Farmland, just a little extra so there's no confusion between Farm.construct and construct_farm